pyproject.toml

```diff
@@ -86,6 +86,7 @@ force-exclude = '''
 ignore-words-list = 'astroid,crasher,asend'
 
 [tool.ruff]
+target-version = "py38"
```

Comment:
Ruff is supposed to auto-detect your target version from the `requires-python = ">=3.8"`. Is this no longer the case?

Reply:
Didn't know that was a thing. I'll remove it.